### ISSUE TITLE
Create Mod Incompatibility

### ISSUE THREAD

**Issue by CellCitizen** [LunaPixelStudios/SteamPunk#121]
### Create Mod Issues

When using the create mod and making moving contraptions such as farms, deployers do not work while moving. After searching for solutions it seems that the primary reason for this is on account of an FTB Teams or other compatibility. The consensus on message boards seems to be that its because of deployers being detected as "fake players" on claimed chunks with the ftbteams/ftbchunks mod. The only issue is that all of the FTB mods seems to be lacking in many basic features on this version. There is no way to allow fake players in the team settings, even in singleplayer or when given OP. As well, even when not in a team or claiming chunks it seems to still not work, so I'm not sure

This essentially destroys any options for automated farms using deployers which is the most common and easy way to do them.

TLDR; Saplings cant be placed from moving deployers in farms

### Reproduction Steps

1. Build sapling farm with deployer (schematic example: https://www.mediafire.com/file/tykx2l6mxqqxp1t/basic_tree_farm.nbt/file)
2. Turn on farm

### Operating System

Windows 10

### Minecraft Version

1.19

### Modloader

Forge

### Modpack Version

v21.5

### Optifine

No

**Comment by chorbintime** [comment]
please report this to the FTB developers
